packages/x-charts-vendor/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@mui/x-charts-vendor",
+  "version": "7.8.0",
+  "description": "Vendored dependencies for MUI X Charts",
+  "author": "MUI Team",
+  "main": "./index.js",
```

Comment:
This file doesn't exist:

<img width="403" alt="SCR-20240903-bwyo" src="https://github.com/user-attachments/assets/eb5e5d5b-5c67-4a07-801a-021b4ccc5334">

https://publint.dev/@mui/x-charts-vendor@7.15.0

```suggestion
```